docker-compose.arm64.yml

```diff
@@ -1,39 +1,178 @@
+            functions_provider_url: "http://faas-swarm:8080/"
+            read_timeout:  "305s"        # Maximum time to read HTTP request
+            write_timeout: "305s"        # Maximum time to write HTTP response
+            upstream_timeout: "300s"     # Maximum duration of upstream function call - should be more than read_timeout and write_timeout            dnsrr: "true"  # Temporarily use dnsrr in place of VIP while issue persists on PWD
+            faas_nats_address: "nats"
+            faas_nats_port: 4222
+            direct_functions: "true"    # Functions are invoked directly over the overlay network
+            direct_functions_suffix: ""
+            basic_auth: "false"
+            secret_mount_path: "/run/secrets/"
+            scale_from_zero: "false"
+        deploy:
+            resources:
+                # limits:   # uncomment to enable limits
+                #     memory: 100M
+                reservations:
+                    memory: 100M
+            restart_policy:
+                condition: on-failure
+                delay: 5s
+                max_attempts: 20
+                window: 380s
+            placement:
+                constraints:
+                    - 'node.platform.os == linux'
+
+    # Docker Swarm provider
+    faas-swarm:
+        image:  openfaas/faas-swarm:0.5.0-arm64
+        volumes:
+            - "/var/run/docker.sock:/var/run/docker.sock"
+        networks:
+            - functions
+        environment:
+            read_timeout:  "305s"   # set both here, and on your functions
+            write_timeout: "305s"   # set both here, and on your functions
+            DOCKER_API_VERSION: "1.30"
+            basic_auth: "false"
         deploy:
             placement:
                 constraints:
                     - 'node.role == manager'
                     - 'node.platform.os == linux'
+            resources:
+                # limits:   # uncomment to enable limits
+                #     memory: 100M
+                reservations:
+                    memory: 100M
+            restart_policy:
+                condition: on-failure
+                delay: 5s
+                max_attempts: 20
+                window: 380s
+
+    nats:
+        image: nats-streaming:0.11.2
+        # Uncomment the following port mappings if you wish to expose the
+        # NATS client and/or management ports you must also add `-m 8222` to the command
+        # ports:
+        #     - 4222:4222
+        #     - 8222:8222
+        command: "--store memory --cluster_id faas-cluster"
+        networks:
+            - functions
+        deploy:
+            resources:
+                limits:
+                    memory: 125M
+                reservations:
+                    memory: 50M
+            placement:
+                constraints:
+                    - 'node.platform.os == linux'
+
+    queue-worker:
+        image: openfaas/queue-worker:0.5.4-arm64
```

Comment:
I got the following error:

```
docker pull openfaas/queue-worker:0.5.4-arm64
Error response from daemon: manifest for openfaas/queue-worker:0.5.4-arm64 not found
```

Should we merge or wait until I can publish the image?